protocols/https-1way-elytron/pom.xml

```diff
@@ -0,0 +1,116 @@
+
+    <dependencies>
+        <dependency>
+            <groupId>io.thorntail</groupId>
+            <artifactId>elytron</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.thorntail</groupId>
+            <artifactId>undertow</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.thorntail</groupId>
+            <artifactId>arquillian</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>fluent-hc</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+            </resource>
+        </resources>
+
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>keytool-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>server-keystore</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>generateKeyPair</goal>
+                        </goals>
+                        <configuration>
+                            <keystore>${project.build.directory}/server-keystore.jks</keystore>
+                            <storepass>server-password</storepass>
+                            <alias>server-key</alias>
+                            <keypass>server-password</keypass>
+                            <dname>CN=localhost</dname>
+                            <sigalg>SHA1withRSA</sigalg>
+                            <validity>100</validity>
+                            <keyalg>RSA</keyalg>
+                            <keysize>2048</keysize>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>server-certificate</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>exportCertificate</goal>
+                        </goals>
+                        <configuration>
+                            <keystore>${project.build.directory}/server-keystore.jks</keystore>
+                            <storepass>server-password</storepass>
+                            <alias>server-key</alias>
+                            <keypass>server-password</keypass>
```

Comment:
keypass in <goal>exportCertificate</goal> is unused. There are 12 occurrences (IDEA identifies it as "Element keypass is not allowed here").

Reply:
It's unfortunate that `keytool` accepts `-keypass` here, but the Maven plugin doesn't expose that option. I'll remove it.